packages/apputils/src/dialog.tsx

```diff
@@ -97,6 +97,15 @@ export class Dialog<T> extends Widget {
         return renderer.createButtonNode(button);
       })
     );
+    this._checkboxNode = normalized.checkbox
+      ? renderer.createCheckboxNode({
+          label: normalized.checkbox.label ?? '',
+          caption:
+            normalized.checkbox.caption ?? normalized.checkbox.label ?? '',
+          checked: normalized.checkbox.checked ?? false,
+          className: normalized.checkbox.className ?? ''
+        })
+      : null;
```

Comment:
There are a lot of duplicate references here. Is it possible to use destructuring along with defaults?
```ts
const checkbox = normalized.checkbox
if(checkbox) {
    const { label = '', caption = '', checked = false, className = '' } = checkbox
    this._checkboxNode = renderer.createCheckboxNode({ 
        label, 
        caption: caption ?? label, 
        checked, 
        className 
    })
} else {
    this._checkboxNode = null
}
```